File: Bridge Python Client/metamenth/enumerations/sensor_measure.py
from enum import Enum


class SensorMeasure(Enum):
    """
    Various phenomena that can be measured by a sensor

    Author: Peter Yefi
    Email: peteryefi@gmail.com
    """
    TEMPERATURE = "Temperature"
    PRESSURE = "Pressure"
    GAS_VELOCITY = "GasVelocity"
    LIQUID_VELOCITY = "LiquidVelocity"
    LUMINANCE = "Luminance"
    CARBON_DIOXIDE = "CarbonDioxide"
    NOISE = "Noise"
    CURRENT = "Current"
    VOLTAGE = "Voltage"
    GLOBAL_RADIATION = "GlobalRadiation"
    DIRECT_RADIATION = "DirectRadiation"
    SMOKE = "Smoke"
    OCCUPANCY = "Occupancy"
    DAYLIGHT = "Daylight"
    AIR_VOLUME = "AirVolume"
    HUMIDITY = "Humidity"
    OTHER = "Other"

File: Bridge Java Server/src/main/java/com/middleware/enums/SensorMeasure.java
package com.middleware.enums;

public enum SensorMeasure {
    TEMPERATURE("Temperature"),
    PRESSURE("Pressure"),
    GAS_VELOCITY("GasVelocity"),
    LIQUID_VELOCITY("LiquidVelocity"),
    LUMINANCE("Luminance"),
    CARBON_DIOXIDE("CarbonDioxide"),
    NOISE("Noise"),
    CURRENT("Current"),
    VOLTAGE("Voltage"),
    GLOBAL_RADIATION("GlobalRadiation"),
    DIRECT_RADIATION("DirectRadiation"),
    SMOKE("Smoke"),
    OCCUPANCY("Occupancy"),
    DAYLIGHT("Daylight"),
    AIR_VOLUME("AirVolume"),
    HUMIDITY("Humidity"),
    OTHER("Other");

    private final String value;

    SensorMeasure(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}
